tuna/example/tables.py

```diff
@@ -34,17 +36,17 @@
 class ExampleDBTables(DBTablesInterface):
   """Represents db tables for example lib"""
 
-  def __init__(self, **kwargs):
+  def __init__(self, **kwargs: Dict[str, Any]) -> None:
     """Constructor"""
     super().__init__(**kwargs)
 
     #for pylint
-    self.job_table = None
-    self.session_table = SessionExample
+    self.job_table: Job = None
```

Comment:
Why not set `self.job_table: Job = Job` right here and remove self.set_tables()?

Reply:
Perhaps this example is overly trivial. I think Alex wrote it this way to illustrate that these tables could change depending on another variable. In miopen these tables switch depending on the config type.